liveHomeFrontend/src/components/Label.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import styles from '../assets/styles/components/Label.module.sass';
+
+const Label = ({ value }) => {
+  return (
+    <>
+      <label className={styles.inputLabel}>{value}</label>
+    </>
+  );
+};
+
+export default Label;
```

Comment:
This is the label for inputs?

Reply:
this component is not necessary it was already created with the inputs